templates/index.html

```diff
@@ -8,250 +8,184 @@
+        最新のニュースとコンテンツ
+      </a>
+    </h2>
+    <div>
+      <div class="col-4">
+        <h3 class="p-heading--four"><a href="https://blog.ubuntu.com/2018/01/30/ntt-technocross-becomes-canonical-certified-support-partner-in-japan">NTT TechnoCross becomes Canonical Certified Support Partner in Japan</a></h3>
+        <p><time pubdate="" datetime="2018-06-18T14:25:58">30 January 2018</time></p>
+      </div>
 
-        <p><a href="https://www.ubuntu.com/cloud/contact-us">Contact us to talk about your cloud needs&nbsp;&rsaquo;</a></p>
+      <div class="col-4">
+        <h3 class="p-heading--four"><a href="https://blog.ubuntu.com/2017/12/12/canonical-helps-dena-lower-operational-cost-of-always-on-service" >Canonical helps DeNA lower operational cost of always-on service</a></h3>
+        <p><time pubdate="" datetime="2018-06-18T08:09:52">12 December 2017</time></p>
       </div>
-      <div class="col-6">
-        <div class="cloud-tools">
-          <div class="cloud-tools__list">
-            <a class="cloud-tools__item item-one" title="OpenStack">OpenStack</a>
-            <span class="cloud-tools__item item-line item-one-line"></span>
-            <a class="cloud-tools__item item-two" title="BootStack">BootStack</a>
-            <span class="cloud-tools__item item-line item-two-line"></span>
-            <a class="cloud-tools__item item-three" title="Juju">Juju</a>
-            <span class="cloud-tools__item item-line item-three-line"></span>
-            <a class="cloud-tools__item item-four" title="MAAS">MAAS</a>
-            <span class="cloud-tools__item item-line item-four-line"></span>
-            <a class="cloud-tools__item item-five" title="Autopilot">Autopilot</a>
-            <span class="cloud-tools__item item-line item-five-line"></span>
-            <a class="cloud-tools__item item-six" title="Public cloud">Public cloud</a>
-            <span class="cloud-tools__item item-line item-six-line"></span>
-            <span class="cloud-tools__item item-seven">Ubuntu</span>
-          </div>
-        </div>
+
+      <div class="col-4">
+        <h3 class="p-heading--four"><a href="https://pages.ubuntu.com/OpenStack_made_easy_jp.html">Openstack made easy</a></h3>
+        <p><time pubdate="" datetime="2018-06-06T13:00:35">25 December 2017</time></p>
       </div>
     </div>
   </div>
-  <!-- /.row -->
+</div>
+</section>
 
-  <!-- Section 3: Build your own cloud with Ubuntu OpenStack -->
-  <div class="p-strip--accent is-dark">
-    <div class="row">
+
+
+<section class="p-strip is-deep u-image-position is-bordered u-vertically-center">
+  <div class="row">
+    <div class="col-6">
+      <img src="{{ ASSET_SERVER_URL }}0bfbd13b-cloud_section_bundle.svg" width="50%" class="u-image-position--top u-image-position--left u-hide--small" alt=""/>
```

Comment:
I think remove ,u-image-position--top ? looks better

Reply:
thanks, agreed to leave for now.